ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -61,3 +61,16 @@ extension View {
                               shouldScale: shouldScale))
     }
 }
+
+extension Locale {
+    /// Determines if layout direction of current language is `.leftToRight`.
+    /// - Returns: True if layout direction is left to right, false otherwise.
+    func isLeftToRightLayoutDirection() -> Bool {
```

Comment:
supernit: I can be swayed either direction(no pun intended) but there are more api that check RightToLeft than LeftToright in fluent.

Reply:
True, let me adjust!